scss/civicrm/common/_notifications.scss

```diff
@@ -10,6 +10,11 @@
       h1::before {
         @include fa-icon($font-size-base, $fa-var-alert, $brand-danger);
       }
+
+      // This removes an extra icon on the body of error notifications:
```

Comment:
Where was this icon coming from?

Reply:
CiviCRM. Tried to find it, but it seems to come from core validation.

The first icon (exclamation circle) comes from Shoreditch ([source](https://github.com/compucorp/org.civicrm.shoreditch/blob/33b3b4da32f06af893da2df1e6b359faad957668/scss/civicrm/common/_notifications.scss#L11))

Reply:
@reneolivo so does it mean that all this time we had two icons whenever shoreditch was enabled? seems very weird to me

Reply:
yes, but this doesn't happen in all cases. For example I have a notification for verifying max file upload size, and that one looks fine. This seems to me like the validation scenario was not taken into consideration.

![download 35](https://user-images.githubusercontent.com/1642119/30591029-b5ae00aa-9ced-11e7-999a-81fa74b826c4.png)